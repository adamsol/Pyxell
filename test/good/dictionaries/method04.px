
d = {for c in 'a'..'z' yield c: c.code}

d.remove('b')
print d.get('b')

d.remove('A')
print d.length

d': {Int:Char}
for _ in 1..d.length do
    p = d.pop()
    d'[p.b] = p.a

print d.length
print d' == {for c in 'a'..'z' if c != 'b' yield c.code: c}

d'.clear()
print d'
