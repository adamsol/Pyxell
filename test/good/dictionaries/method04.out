null
25
0
true
{:}
